packages/opds-web-client/src/components/BorrowButton.tsx

```diff
@@ -1,15 +1,13 @@
 import * as React from "react";
 
-export interface BorrowButtonProps extends React.HTMLProps<any> {
+export interface BorrowButtonProps extends React.HTMLProps<{}> {
   borrow: () => Promise<any>;
 }
 
-export default class BorrowButton extends React.Component<BorrowButtonProps, void> {
+export default class BorrowButton extends React.Component<BorrowButtonProps, {}> {
   render() {
-    let props = Object.assign({}, this.props);
-    delete props["book"];
-    delete props["borrow"];
-    delete props["key"];
+    const { ref, borrow, children, ...rest } = this.props;
+    let props = Object.assign({}, rest);
```

Comment:
The spread operator already makes a new object so I don't think there's any need for the Object.assign.

Reply:
Gotcha, will update the three cases where I used this technique.